File: cli/src/main/java/com/gmail/marcosav2010/command/base/ExitCMD.java
package com.gmail.marcosav2010.command.base;

import com.gmail.marcosav2010.command.Command;

class ExitCMD extends Command {

    ExitCMD() {
        super("exit", new String[]{"e"});
    }

    @Override
    public void execute(String[] arg, int args) {
        System.exit(0);
    }
}

File: cli/src/main/java/com/gmail/marcosav2010/command/base/ExitCMD.kt
package com.gmail.marcosav2010.command.base

import com.gmail.marcosav2010.command.Command
import kotlin.system.exitProcess

internal class ExitCMD : Command("exit", arrayOf("e")) {

    override fun execute(arg: Array<String>, length: Int) {
        exitProcess(0)
    }
}